Code/Framework/AzCore/AzCore/Debug/Profiler.h

```diff
@@ -318,6 +52,40 @@ namespace AZStd
 
 namespace AZ
 {
+    class ProfileScope
+    {
+    public:
+        static uint32_t GetSystemID(const char* system);
+
+        template<typename... T>
+        static void BeginRegion([[maybe_unused]] const char* system, [[maybe_unused]] char const* eventName, [[maybe_unused]] T const&... args)
```

Comment:
Nitpick: East- and west ending is mixed here. O3DE is on team West-end for the most part. (So `const T&`.)